app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -26,18 +26,18 @@ const RATE_LIMIT_TYPES = {
  * default is 'RATE_LIMITED'
  */
 const RATE_LIMIT_MAP = {
-  [MESSAGE_TYPE.ETH_SIGN]: RATE_LIMIT_TYPES.NON_RATE_LIMITED,
```

Comment:
All of these should be `MESSAGE_TYPE` as it's concerning which JSON-RPC methods are limited.

Reply:
Same as above, all reverted.